modules/store/src/feature_creator.ts

```diff
@@ -138,7 +138,7 @@ export function createFeature<FeatureName extends string, FeatureState>(
  *   name: 'products',
  *   reducer: createReducer(
  *     initialState,
- *     on(ProductsApiActions.loadSuccess(state, { products }) => ({
+ *     on(ProductsApiActions.loadSuccess, (state, { products }) => ({
  *       ...state,
  *       products,
  *     }),
```

Comment:
Thanks @DavidGoestemeier! Can you also add a missing closing bracket:

```suggestion
 *     })),
```

Reply:
Done